app/imports/components/workers/RemoveWorker.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+
+import { removeWorker } from '/imports/api/usersMethods.js';
+
+export class RemoveWorker extends Component {
+  constructor(props) {
+    super(props);
+
+    this.onRemoveWorker = this.onRemoveWorker.bind(this);
+  }
+
+  onRemoveWorker() {
+    removeWorker.call({
+      workerId: this.props.workerId,
+    });
+  }
+
+  render() {
+    return (
+      <button className="pt-button pt-large" onClick={this.onRemoveWorker}>
```

Comment:
I feel like this button needn't be large, but we just need a convention across the code